src/config/2019.json

```
@@ -1056,6 +1056,16 @@
       "website": "https://data-seo.com",
       "github": "voltek62"
     },
+    "hakaCode": {
+      "name": "William Sandres",
+      "teams": [
+        "Translators",
+      ],
+      "avatar_url": "https://avatars0.githubusercontent.com/u/33907565?v=4&s=200",
+      "website": "hakacode.github.io",
```

Without this it's a relative link:

```suggestion
      "website": "https://hakacode.github.io",
```